js/tabs/js/lumx.tabs_directive.js

```diff
@@ -75,6 +76,184 @@ angular.module('lumx.tabs', [])
             }
         };
 
+        function isPaginationActive() 
+        {
+            var tabsWidth = links.outerWidth();
+            var tabsVisibleWidth = links.parent('.tabs').outerWidth();
+
+            return tabsWidth > tabsVisibleWidth;
+        }
+
+        function getFirstHiddenLeftTab()
+        {
+            var linkContainer = links.parent('.tabs');
+            var leftBorderContainer = linkContainer.offset().left;
+            var tabTags = links.find('.tabs-link');
+
+            var firstTabHidden;
+
+            for (var i = 0; i < tabTags.length; i++) 
+            {
+                var leftTab = $(tabTags[i]).offset().left;
+
+                if (!firstTabHidden && leftTab > (leftBorderContainer - linkContainer.outerWidth()) && leftTab < leftBorderContainer) 
+                {
+                    firstTabHidden = tabTags[i];
+                    break;
+                }
+            }
+
+            return firstTabHidden;
+        }
+
+        function getFirstHiddenRightTab()
+        {
+            var linkContainer = links.parent('.tabs');
+            var rightBorderContainer = linkContainer.offset().left + linkContainer.outerWidth();
+            var tabTags = links.find('.tabs-link');
+
+            var firstTabHidden;
+
+            for (var i = 0; i < tabTags.length; i++) 
+            {
+                var rightTab = $(tabTags[i]).offset().left + $(tabTags[i]).outerWidth();
+
+                if (!firstTabHidden && rightTab > rightBorderContainer) 
+                {
+                    firstTabHidden = tabTags[i];
+                    break;
+                }
+            }
+
+            return firstTabHidden;
+        }
+
+        function getFirstVisibleTab()
+        {
+            var linkContainer = links.parent('.tabs');
+            var leftBorderContainer = linkContainer.offset().left;
+            var tabTags = links.find('.tabs-link');
+
+            var firstTabVisible;
+
+            for (var i = 0; i < tabTags.length; i++)
+            {
+                var leftBorderTab = $(tabTags[i]).offset().left;
+                if (!firstTabVisible && leftBorderTab > leftBorderContainer)
+                {
+                    firstTabVisible = tabTags[i];
+                    break;
+                }
+            }
+
+            return firstTabVisible;
+        }
+
+        function isPaginationLeftDisabled () 
```

Comment:
This was my first try but I had this error :  ```Error: [$parse:isecdom] Referencing DOM nodes in Angular expressions is disallowed! Expression: !getFirstHiddenLeftTab()```
That's the reason for writing ```isPaginationLeftDisabled``` and ```isPaginationRightDisabled```